### PR TITLE
Dev Thanakorn

### PR DESCRIPTION
**Pull request by tnktian**
This is a new feature that I wrote for make our program better.

### CODE REVIEW

**Comment by arthasem** [comment]
<html>
<body>


(つ≧▽≦)つ
--



</body>
</html>

File: lab-exercise-1/ThanakornFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class ThanakornFeature {
+     public void feature(){
+         // this is a new feature
+     }
```
Comment (nickttrps) [inline lab-exercise-1/ThanakornFeature.java:4]: loan za pa kun puen.

File: lab-exercise-1/ThanakornFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class ThanakornFeature {
+     public void feature(){
+         // this is a new feature
```
Comment (sRubyy) [inline lab-exercise-1/ThanakornFeature.java:3]: Well done la bro ^^

File: lab-exercise-1/ThanakornFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+public class ThanakornFeature {
```
Comment (plphxwxt) [inline lab-exercise-1/ThanakornFeature.java:1]: You think you who huh ! what you do man

Comment (YunduoLuo) [inline lab-exercise-1/ThanakornFeature.java:1]: nice work dude:)

File: lab-exercise-1/ThanakornFeature.java [hunk 4]
```diff
@@ -1,5 +1,8 @@
 public class ThanakornFeature {
      public void feature(){
          // this is a new feature
+         int x = 0;
+          x = x+1;
```
Comment (nickttrps) [inline lab-exercise-1/ThanakornFeature.java:5]: clean bro.

**Review by sRubyy — COMMENTED** [review]
cleann

**Review by plphxwxt — COMMENTED** [review]
So smart

**Review by YunduoLuo — COMMENTED** [review]
nice.

File: lab-exercise-1/ThanakornFeature.java (hunk 4 above)
Comment (arthasem) [inline lab-exercise-1/ThanakornFeature.java:5]: nice one!!!!

**Pull request merged**